keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -63,67 +63,61 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
 ## Summary
 
-We will add support in kubelet for the `pullIfNotPresent` image pull policy, for
-ensuring images pulled with pod imagePullSecrets are re-authenticated for other
-pods that do not have the same imagePullSecret/auths used to successfully pull
-the images in the first place.
+Give the admin the ability to ensure pods that use an image are authorized to access that image. This will culminate in changes to the `IfNotPresent` and
+`Never` pull policies, as the `Always` policy will go through an authentication check each time.
 
-This policy change will have no affect on the `pull always` image pull
-policy or for images that are preloaded.
+When this feature is enabled, kubelet will check against an internal cache that maps images to credentials used. If the image has not been successfully pulled with the given credentials
```

Comment:
All the information from here to the end of this section looks like it belongs to the proposal and/or implementation details sections.

Reply:
updated a bit to be less implementation detail heavy